src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();

```

Comment:
```suggestion
    private RegistrationService registrationService;
    
    @BeforeEach
    void setUp() {
        registrationService = new RegistrationServiceImpl();
    }

```

See [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/junit/registration-validation)